src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Hakemus/hakemusContainer.js

```diff
@@ -185,6 +185,11 @@ const HakemusContainer = props => {
         return changeObjects;
       };
 
+      let perustelutLiiitteetChanges =
+        R.path(
+          ["meta", "tutkinnotjakoulutuksetLiitteet", "changeObjects"],
+          muutospyynnot.muutospyynto
+        ) || [];
```

Comment:
Variable name could be more specific. The code is fetching the attachments of the Tutkinnot ja koulutukset section but the variable (perustelutLiitteetChanges) doesn't indicate it.